sdk/keyvault/Azure.Security.KeyVault.Shared/src/KeyVaultPipeline.cs

```diff
@@ -171,115 +190,135 @@ public Page<T> GetPage<T>(Uri firstPageUri, string nextLink, Func<T> itemFactory
             where TContent : IJsonSerializable
             where TResult : IJsonDeserializable
         {
-            using Request request = CreateRequest(method, path);
-            request.Content = RequestContent.Create(content.Serialize());
+            using HttpMessage message = CreateMessage(method, null, path);
```

Comment:
Does this call, passing in null for the RequestContext, have good test coverage?